src/main/java/frc/utils/alerts/Alert.java

```diff
@@ -0,0 +1,36 @@
+package frc.utils.alerts;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Timer;
+import frc.utils.DriverStationUtils;
+import org.littletonrobotics.junction.Logger;
+
+public class Alert {
+
+	public enum AlertType {
+
+		ERROR,
+		PLACEHOLDER,
```

Comment:
why is PLACEHOLDER still here? if you want to keep it you should change the name